### PR TITLE
Bootstrap carousel

### PR DESCRIPTION
**Pull request by amritsiidhu**
Now uses Bootstrap Carousel over the slick carousel
Has kept the same functionality and displays images the same
This also fixes the resizing of the carousel caption

### CODE REVIEW

File: app/static/css/style.css [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 }
 /*---Firefox Bug Fix --*/
-.carousel-item {
+.ousel-item {
```
Comment (kenlt-uk) [inline app/static/css/style.css]: Is this class needed? as I don't think it's needed

**Comment by kenlt-uk** [comment]
The code needs to be rebased.

Also I noticed that the carousel wasn't resizing height wise, which also means that the text is not changing, is this intentional?

File: app/static/css/style.css (hunk 1 above)
Comment (AmritSidhu) [reply app/static/css/style.css]: This was default css when implementing bootstrap. I don't believe it makes a difference but was recommended to keep

Comment (AmritSidhu) [reply app/static/css/style.css]: I also left the carousel text the same size to keep consistency with the rest of the page as no other text resizes. I can implement it to change size at certain page width if you preferred?

Comment (amritsiidhu) [reply app/static/css/style.css]: This was default css when implementing bootstrap. I don't believe it makes a difference but was recommended to keep.
I have for now fixed the missing characters

Comment (amritsiidhu) [reply app/static/css/style.css]: I also left the carousel text the same size to keep consistency with the rest of the page as no other text resizes. I can implement it to change size at certain page width if you preferred?

Comment (kenlt-uk) [reply app/static/css/style.css]: I'll check with others whether they expect the image to resize but for now lets leave it as it is

**Pull request merged**